### PR TITLE
[Bulk] Validation: Fix missing alt text 3/17 part 2

### PR DESCRIPTION
**Pull request by RedZephyr13**
This PR fixes 96 missing alt text issues.

### CODE REVIEW

File: iis/configuration/system.webServer/security/index.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ The `<security>` element is included in the default installation of IIS 7.
 3. Scroll to the **Security** section in the **Home** pane, and then double-click **Authentication**.
 4. In the **Authentication** pane, select **Basic Authentication**, and then, in the **Actions** pane, click **Enable**.
 5. In the **Authentication** pane, select **Anonymous Authentication**, and then click **Disable** in the **Actions** pane.  
-    [![](index/_static/image8.png)](index/_static/image7.png)
+    [![Screenshot that shows the Authentication pane.  Anonymous Authentication is selected. The Actions pane lists Disabled and Edit.](index/_static/image8.png)](index/_static/image7.png)
```
Comment (LizCasey1) [inline iis/configuration/system.webServer/security/index.md:123]: There's an extra space before the word Anonymous. : )

File: iis/web-hosting/joining-the-web-hosting-gallery/validate-your-server-configuration.md [hunk 2]
```diff
@@ -158,10 +158,10 @@ This Validator uses WMI to check if a product is in Win32\_Products. It's an alt
 1. Start Visual Studio
 2. Create *.NET 2.0*, C# class library project:  
   
-    ![](validate-your-server-configuration/_static/image3.png)
+    ![Screenshot that shows the Add New Project window.  Class Library is selected.](validate-your-server-configuration/_static/image3.png)
```
Comment (LizCasey1) [inline iis/web-hosting/joining-the-web-hosting-gallery/validate-your-server-configuration.md:161]: Extra space in front of "Class". : )

**Comment by RedZephyr13** [comment]
@Rick-Anderson @mcleblanc

This PR fixes validation warnings/suggestions per a platform-wide push to increase content health scores. Please review at your convenience and sign off or merge if you approve or let me know if I need to make any other changes.

CC: @MonicaRush @snoviking

File: iis/configuration/system.applicationHost/sites/site/application/virtualDirectory.md [hunk 3]
```diff
@@ -55,10 +55,10 @@ The `<virtualDirectory>` element is included in the default installation of IIS
         - On the taskbar, click **Start**, and then click **Control Panel**.
         - Double-click **Administrative Tools**, and then double-click **Internet Information Services (IIS) Manager**.
 2. In the **Connections** pane, expand the server name, expand **Sites**, expand the Web site to which you want to add the virtual directory, and then click the application to which you want to add the virtual directory.  
-    [![](virtualDirectory/_static/image2.png)](virtualDirectory/_static/image1.png)
+    [![Screenshot that shows the My Application Home pane.](virtualDirectory/_static/image2.png)](virtualDirectory/_static/image1.png)
```
Comment (Rick-Anderson) [inline iis/configuration/system.applicationHost/sites/site/application/virtualDirectory.md:58]: This crazy syntax was used [originally](https://github.com/MicrosoftDocs/iis-docs/blob/56eab07132bd12f69affa480fefc83aab45a06a7/iis/configuration/system.applicationHost/sites/site/application/virtualDirectory.md) as show [here](https://github.com/MicrosoftDocs/iis-docs/blob/56eab07132bd12f69affa480fefc83aab45a06a7/iis/configuration/system.applicationHost/sites/site/application/virtualDirectory.md?plain=1#L63,#L66)

**Review by Rick-Anderson — APPROVED** [review]

File: iis/configuration/system.webServer/security/index.md (hunk 1 above)
Comment (Rick-Anderson) [reply iis/configuration/system.webServer/security/index.md]: Nice catch, I'm not that picky. Have you ever been to Dishman Dodge?

File: iis/configuration/system.applicationHost/sites/siteDefaults/ftpServer/security/commandFiltering/add.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 author: rick-anderson
-description: "Overview The &lt;add&gt; element adds a unique entry to the collection of FTP commands that the FTP service will allow or deny. Note : Special attention must..."
+description: "Overview The &lt;add&gt; element adds a unique entry to the collection of FTP commands that the FTP service will allow or deny. "
 ms.date: 09/26/2016
 ms.assetid: 953b3dbd-2f08-40e3-bc67-3516848bb238
 msc.legacyurl: /configreference/system.applicationhost/sites/sitedefaults/ftpserver/security/commandfiltering/add
 msc.type: config
 ---
-# Adding FTP Command Filtering Entries &lt;add&gt;
+# Adding FTP Command Filtering Entries \<add>
```
Comment (Rick-Anderson) [inline iis/configuration/system.applicationHost/sites/siteDefaults/ftpServer/security/commandFiltering/add.md:10]:
```suggestion
# Add FTP Command Filtering Entries \<add>
```
Remove gerund  and fix warning

**Comment by Rick-Anderson** [comment]
@RedZephyr13 @LizCasey1  your work is done here, I'm experimenting with some of the other warnings. I'll S&M soon.

**Comment by Rick-Anderson** [comment]
@RedZephyr13 @LizCasey1  I got rid of all the remaining warnings. I'm guessing you're working on specific warnings and not files so normally the non alt-text warnings would be done in another pass.

File: iis/configuration/system.applicationHost/sites/site/application/virtualDirectory.md (hunk 3 above)
Comment (RedZephyr13) [reply iis/configuration/system.applicationHost/sites/site/application/virtualDirectory.md:58]: Yes, there's two images.  One thumbnail image and a normal image.

**Comment by RedZephyr13** [comment]
@Rick-Anderson - Yes, we're working on specific validation warnings.

**Pull request merged**